can-route.md

```diff
@@ -92,12 +92,12 @@ You can listen to changes in a map with `on(eventName, handler(ev, args...))` an
 ### Listening to changes in can-route
 
 Listen to changes in history by [can-event.addEventListener listening] to
-changes in can-route like:
+changes of can-route's `matched` compute:
 
 ```js
-route.on('foo', function(ev, attr, how, newVal, oldVal) {
-	// Foo changed!
-})
+route.matched.on('change', function(ev, attr, how, newVal, oldVal) {
```

Comment:
I guess the big question is do we want to teach people they can listen to `__url`?  Maybe no for now.

Reply:
Leaving it undocumented is fine for now.  We can always decide to make it official later.